esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3014,7 +3014,11 @@ int WUSchedule::run()
     unsigned int waitTimeMillies = 1000*60;
     while(!stopping)
     {
-        if (!detached)
+        if (!m_container)
+        {
+            WARNLOG("WSWorkuinits schedule is waiting for container being set.");
```

Comment:
I'm not even sure this is a warning.  Since it's probably a timing thing this could be debug info?

Reply:
Also there's a typo in that warning text